impeller/entity/contents/filters/filter_contents.cc

```diff
@@ -176,6 +176,16 @@ std::shared_ptr<FilterContents> FilterContents::MakeSrgbToLinearFilter(
   return filter;
 }
 
+std::shared_ptr<FilterContents> FilterContents::MakeComposeImageFilter(
+    FilterInput::Ref input,
+    ImageFilterProc outer_proc,
+    ImageFilterProc inner_proc,
+    const Matrix& effect_transform) {
+  auto contents = inner_proc(input, effect_transform);
+  contents = outer_proc(FilterInput::Make(contents), effect_transform);
+  return contents;
+}
```

Comment:
Since chaining filters is already supported as part of the public API via `FilterInput`, let's just remove this factory function and do this chaining directly in the dispatcher.

Reply:
Done